assets/js/src/newsletter_editor/utils.js

```diff
@@ -0,0 +1,17 @@
+const isEventInsideElement = (event, $el) => {
```

Comment:
You can directly do `export const ...` without the need of the export statement below.